apollo-router/src/plugins/subscription.rs

```diff
@@ -182,6 +213,96 @@ pub(crate) struct WebSocketConfiguration {
     pub(crate) protocol: WebSocketProtocol,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Deserialize, Serialize, JsonSchema)]
+#[serde(deny_unknown_fields, default)]
+pub(crate) struct SseConfiguration {
+    /// Enable configuration
+    #[serde(default = "default_sse_config_enabled")]
+    pub(crate) enabled: bool,
+
+    /// Path use to get the SSE stream
+    pub(crate) path: Option<String>,
+
+    /// If `reconnect` is `true`, the client will automatically
+    /// try to reconnect if the stream ends due to an error. If it is `false`  (the [default]),
+    /// the client will stop receiving events after an error.
+    pub(crate) reconnect: Option<bool>,
```

Comment:
As we're using `.unwrap_or_default()` when using for `ReconnectOptions` I suggest to change it to `bool` instead of Option. It will still work because of the serde default.